recipes/pydrift/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pydrift" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 572a5f5ee3cfde052ee858cc2ed741fbb5c3df9fbcd44a70fe1d208b751a163e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
```

Comment:
For pip packages it is sufficient to have the packages listed in the run section.